packages/gatsby-cli/src/create-cli.js

```diff
@@ -346,6 +346,21 @@ module.exports = argv => {
         }
       ),
     })
+    .command({
+      command: `plugin`,
+      desc: `Welcome to the Gatsby source plugin help center!
+
+      Resources:
+- Walk through the tutorial (https://www.gatsbyjs.org/docs/source-plugin-tutorial/)
+- Read requirements for Gatsby packages (file and naming conventions) (https://www.gatsbyjs.org/docs/how-plugins-work/)
+- Submit to plugin library (https://www.gatsbyjs.org/contributing/submit-to-plugin-library/)
+- Join Discord #plugin-authoring channel to ask questions
```

Comment:
We'll need to create this channel before merging this PR too :)

Reply:
It has been done 😉

![image](https://user-images.githubusercontent.com/13099512/56459534-41feb800-6395-11e9-9845-d8971446a814.png)